astropy/wcs/wcs.py

```diff
@@ -1008,10 +1012,10 @@ def _write_distortion_kw(self, hdulist, dist='CPDIS'):
 
         if dist == 'CPDIS':
             d_kw = 'DP'
-            err_kw = 'CPERR'
+            # err_kw = 'CPERR'
```

Comment:
Also not used. But somehow here I commented instead of deleting. I should be consistent, but I want to hear what others say first.

Reply:
Just delete it!